libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackClientWrapper.cs

```diff
@@ -0,0 +1,554 @@
+        public virtual async Task<GroupSetTopicResponse> GroupsSetTopicAsync(string channelId, string topic)
+        {
+            return await _api.GroupsSetTopicAsync(channelId, topic).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Wraps Slack API's GroupsUnarchiveAsync method.
+        /// </summary>
+        /// <param name="channelId">The channel id.</param>
+        /// <returns>A <see cref="GroupUnarchiveResponse"/> representing the response.</returns>
+        public virtual async Task<GroupUnarchiveResponse> GroupsUnarchiveAsync(string channelId)
+        {
+            return await _api.GroupsUnarchiveAsync(channelId).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Wraps Slack API's JoinDirectMessageChannelAsync method.
+        /// </summary>
+        /// <param name="user">The user to join in.</param>
+        /// <returns>A <see cref="JoinDirectMessageChannelResponse"/> representing the response.</returns>
+        public virtual async Task<JoinDirectMessageChannelResponse> JoinDirectMessageChannelAsync(string user)
+        {
+            return await _api.JoinDirectMessageChannelAsync(user).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Wraps Slack API's MarkChannelAsync method.
+        /// </summary>
+        /// <param name="channelId">The channel id.</param>
+        /// <param name="ts">The timestamp.</param>
+        /// <returns>A <see cref="MarkResponse"/> representing the response.</returns>
+        public virtual async Task<MarkResponse> MarkChannelAsync(string channelId, DateTime ts)
+        {
+            return await _api.MarkChannelAsync(channelId, ts).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Wraps Slack API's PostEphemeralMessageAsync method.
+        /// </summary>
+        /// <param name="channelId">The channel id.</param>
+        /// <param name="text">The text of the message.</param>
+        /// <param name="targetuser">The target user to the message.</param>
+        /// <param name="parse">Change how messages are treated.Defaults to 'none'. See https://api.slack.com/methods/chat.postMessage#formatting. </param>
+        /// <param name="linkNames">If to find and link channel names and username.</param>
+        /// <param name="attachments">The attachments, if any.</param>
+        /// <param name="as_user">If the message is being sent as user instead of as a bot.</param>
+        /// <param name="thread_ts">Info about the message coming from a thread. CURRENTLY NOT USED.</param>
+        /// <returns>A <see cref="PostEphemeralResponse"/> representing the response to the message posting.</returns>
+        public virtual async Task<PostEphemeralResponse> PostEphemeralMessageAsync(string channelId, string text, string targetuser, string parse = null, bool linkNames = false, Attachment[] attachments = null, bool as_user = false, string thread_ts = null)
```

Comment:
Replace `targetuser` with `targetUser`, `as_user` with `asUser` and `thread_ts` with `threadTs` to apply the naming conventions of the SDK.